### PR TITLE
Bluetooth communication signoff

### PR DESCRIPTION
**Pull request by CarsonDPope**

### CODE REVIEW

**Review by JesseTNRoberts — COMMENTED** [review]
No time delay constraints seems a problem.

**Review by JesseTNRoberts — COMMENTED** [review]
I am a bit confused why this does not also have to abide by the 1.4 ms time budget. I think I know, and I think it is because you are recalculating filter parameters but executing the filter locally.

At any rate, be sure to make the situation clear.

**Comment by CarsonDPope** [comment]
You are exactly correct, I will work on making that more clear

File: Documentation/Signoffs/Bluetooth_Communication.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+#### Bluetooth Capabilities
+The Arduino UNO R4 WiFi is able to transmit and receive in WiFi and Bluetooth 5 LE<sup>2</sup>. This is accomplished through the ESP32-S3-MINI[3]. Bluetooth 5.0 LE includes a PHY called LE 2M. This can support 2 Mb/s (2,000,000 bits per second). At the application layer, a data rate of up to approximately 1.4 Mb/s (1,400,000 bits per second) is possible[5]. Assuming the max 2 second delay lines of 187.5 kiB [6], each send will be sending 187.5 kiB (or 1,536,000 bits). Therefore, Bluetooth 5 LE  will be able to send a full delay line in 1.10 seconds (1.536 Mb / 1.4 Mbps), and receive the filter index bits (8 bits or less) in 5.714 ms or less<sup>4</sup>. This leaves 0.804 seconds for SPI communication.
+
+spi here
```
Comment (JesseTNRoberts) [inline Documentation/Signoffs/Bluetooth_Communication.md]: I think something is missing?

File: Documentation/Signoffs/Bluetooth_Communication.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+Since the subsystem is able to communicate wirelessly, the physical extent of the system resides at the window preventing any obstruction of pathways<sup>9</sup>. The room's dimensions are 34x24x11 feet. A worst case scenario (lower corner to opposite upper corner) is a distance of 43.05 ft (13.122 meters). The ESP32-S3-MINI has a typical Bluetooth transmit power of ** dBm and a typical receiver sensitivity of ** dBm. Using Bluetooth's range estimation calculator[10], the ESP32-S3-MINI has an estimated range indoors of ** meters, which is greater than 13.122 meters<sup>6</sup>.
```
Comment (JesseTNRoberts) [inline Documentation/Signoffs/Bluetooth_Communication.md]: **?

File: Documentation/Signoffs/Bluetooth_Communication.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+## Buildable schematic
+
+![image](/Documentation/Images/Bluetooth-Communication/Buildable-Schematics/BluetoothCommSchematic.png)  
+
+The above image is a schematic of the Arduino UNO R4 WiFi, the schematic focuses on the most important aspects of the device with respect to the overall design goal. The schematic also shows how the system connects to the main processor subsystem and power management subsystem. 
+
+## Analysis
+
+#### Power
```
Comment (JesseTNRoberts) [inline Documentation/Signoffs/Bluetooth_Communication.md:49]: I'm not sure that this shows the constraint is met?

File: Documentation/Signoffs/Bluetooth_Communication.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+Since the subsystem is able to communicate wirelessly, the physical extent of the system resides at the window preventing any obstruction of pathways<sup>9</sup>. The room's dimensions are 34x24x11 feet. A worst case scenario (lower corner to opposite upper corner) is a distance of 43.05 ft (13.122 meters). The ESP32-S3-MINI has a typical Bluetooth transmit power of 0 dBm (but can be set higher) and a typical receiver sensitivity of -92 dBm. Using Bluetooth's range estimation calculator[10], the ESP32-S3-MINI has an estimated range in an office setting of 14 to 18 meters meters, which is greater than 13.122 meters<sup>5</sup>.
```
Comment (JesseTNRoberts) [inline Documentation/Signoffs/Bluetooth_Communication.md:56]: This is missing a heading

File: Documentation/Signoffs/Bluetooth_Communication.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+
+#### Bluetooth Capabilities
+The Arduino UNO R4 WiFi is able to transmit and receive in WiFi and Bluetooth 5 LE<sup>2</sup>. This is accomplished through the ESP32-S3-MINI[3]. Bluetooth 5.0 LE includes a PHY called LE 2M. This can support 2 Mb/s (2,000,000 bits per second). At the application layer, a data rate of up to approximately 1.4 Mb/s (1,400,000 bits per second) is possible[5]. Assuming the 2 second delay lines of 187.5 kiB [6], each send will be sending 187.5 kiB (or 1,536,000 bits). Therefore, Bluetooth 5 LE  will be able to send a full delay line in 1.10 seconds (1.536 Mb / 1.4 Mbps), and receive the filter index bits (also 187.5 kiB) in 1.10 seconds for a total of 2.20 seconds<sup>4</sup>.
```
Comment (JesseTNRoberts) [inline Documentation/Signoffs/Bluetooth_Communication.md:53]: This is greater than the constraint. Should the constraint be changed? If so, explain why in the constraint section and make the change.

**Review by JesseTNRoberts — COMMENTED** [review]
Same comments

**Review by JesseTNRoberts — APPROVED** [review]
Looks good.

**Pull request merged**